### DISCUSSION TITLE
Support for Bitcoin Core Versions 23/24

### DISCUSSION THREAD

**Discussion by dmullinsgithub** [janoside/btc-rpc-explorer#501] [Q&A] [answered]
Any plans to release a version of btc-rpc-explorer that supports Bitcoin Core 23.x or 24.x?

**Comment by evehash** [comment, marked as answer]
It doest, nope? I'm using it with 24.0.1

**Reply by dmullinsgithub** [reply]
Tried it with 23 a few months ago and had some api issues. Thanks for this info. Will try with 24.0.1.
